src/libs/geotagging.c

```diff
@@ -133,13 +195,698 @@ static gboolean _lib_geotagging_filter_gpx(const GtkFileFilterInfo *filter_info,
+
+  for(GList *i = d->imgs; i; i = g_list_next(i))
+    ((dt_sel_img_t *)i->data)->segid = -1;
+  int total_imgs = 0;
+  int total_pts = 0;
+  line++;
+  for(GList *ts = trkseg; ts; ts = g_list_next(ts))
+  {
+    dt_gpx_track_segment_t *t = (dt_gpx_track_segment_t *)ts->data;
+    gchar *dts = _utc_timeval_to_localtime_text(&t->start_dt, tz_camera, TRUE);
+    gchar *dte = _utc_timeval_to_localtime_text(&t->end_dt, tz_camera, TRUE);
+
+    const int nb_imgs = _count_images_per_track(t, ts->next ? ts->next->data : NULL,self);
+    total_imgs += nb_imgs;
+
+    _set_up_label(t->name, GTK_ALIGN_START, grid, 0, line, PANGO_ELLIPSIZE_NONE);
+    _set_up_label(dte, GTK_ALIGN_START, grid, 1, line, PANGO_ELLIPSIZE_NONE);
+    _set_up_label(dts, GTK_ALIGN_START, grid, 2, line, PANGO_ELLIPSIZE_NONE);
+    char *nb = dt_util_dstrcat(NULL, "%d", t->nb_trkpt);
+    _set_up_label(nb, GTK_ALIGN_CENTER, grid, 3, line, PANGO_ELLIPSIZE_NONE);
+    g_free(nb);
+    nb = dt_util_dstrcat(NULL, "%d", nb_imgs);
+    _set_up_label(nb, GTK_ALIGN_CENTER, grid, 4, line, PANGO_ELLIPSIZE_NONE);
+    g_free(nb);
+    line++;
+    total_pts += t->nb_trkpt;
+    g_free(dts);
+    g_free(dte);
+  }
+
+  char *nb = dt_util_dstrcat(NULL, "%d", total_pts);
+  _set_up_label(nb, GTK_ALIGN_CENTER, grid, 3, line, PANGO_ELLIPSIZE_NONE);
+  g_free(nb);
+  nb = dt_util_dstrcat(NULL, "%d / %d", total_imgs, g_list_length(d->imgs));
+  _set_up_label(nb, GTK_ALIGN_CENTER, grid, 4, line, PANGO_ELLIPSIZE_NONE);
+  g_free(nb);
+
+  g_time_zone_unref(tz_camera);
+  g_free(tz);
+  dt_gpx_destroy(gpx);
+
+  gtk_container_add(GTK_CONTAINER(w), grid);
+
+#ifdef GDK_WINDOWING_QUARTZ
+  dt_osx_disallow_fullscreen(dialog);
+#endif
+  gtk_widget_show_all(dialog);
+  gtk_dialog_run(GTK_DIALOG(dialog));
+
+  gtk_widget_destroy(dialog);
+}
+
+static void _set_selected_images(dt_lib_module_t *self)
+{
+  dt_lib_geotagging_t *d = (dt_lib_geotagging_t *)self->data;
+  if(d->imgs)
+  {
+#ifdef HAVE_MAP
+    _images_images_from_map(self);
+#endif
+    g_list_free_full(d->imgs, g_free);
+  }
+  d->imgs = NULL;
+
+  sqlite3_stmt *stmt;
+  DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
+                              "SELECT imgid FROM main.selected_images",
+                              -1, &stmt, NULL);
+  while(sqlite3_step(stmt) == SQLITE_ROW)
+  {
+    const uint32_t imgid = sqlite3_column_int(stmt, 0);
+    const dt_image_t *cimg = dt_image_cache_get(darktable.image_cache, imgid, 'r');
+    char dt[DT_DATETIME_LENGTH];
+    if(!cimg) continue;
+    memcpy(dt, cimg->exif_datetime_taken, DT_DATETIME_LENGTH);
+    dt_image_cache_read_release(darktable.image_cache, cimg);
+
+    dt_sel_img_t *img = g_malloc0(sizeof(dt_sel_img_t));
+    if(!img) continue;
+    memcpy(img->dt, dt, DT_DATETIME_LENGTH);
+    img->imgid = imgid;
+    d->imgs = g_list_prepend(d->imgs, img);
+  }
+  sqlite3_finalize(stmt);
+//  const GList *imgs = dt_view_get_images_to_act_on(TRUE, TRUE);
```

Comment:
probably a left over of debug ?

Reply:
Correct. Removed.
Images_to_act_on() was the previous solution but creates glitches when moving the mouse around. 
Hovering is great in some circumstances but we should be able to switch it off when not desired.